cmd/kogito/command/install/kogito_operator.go

```diff
@@ -22,10 +22,12 @@ import (
 )
 
 type installKogitoOperatorFlags struct {
-	namespace        string
-	image            string
-	installDataIndex bool
-	force            bool
+	namespace          string
+	image              string
+	installDataIndex   bool
+	installJobsService bool
+	installMgmtConsole bool
```

Comment:
missing installAll ?

Reply:
I thought about that, but I don't think we should add it in here. Won't be a little weird from UX perspective? 

```
kogito install operator --install-all-services
```
IDK. Again, I think we should bring UX team to these discussions and review the entire CLI commands.

Reply:
agreed. but for know, I think we should just have consistency :)
`--install-all-services` could be good

Reply:
OK, I'll add it.